testData/solutions/linux.json

```diff
@@ -1430,15 +1430,61 @@
             "configuration": {
                 "type": "gpii.xrandr",
                 "capabilities": [
-                    "display.screenEnhacements",
+                    "display.screenEnhancement",
                     "applications.org\\.freedesktop\\.xrandr.id"
-                ]
-           }
+                ],
+                "capabilitiesTransformations": {
+                    "screen-resolution": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "inputPath": "http://registry\\.gpii\\.net/common/screenResolution",
+                            "defaultInputValue": "normal",
+                            "options": {
+                                "high": {
+                                    "outputValue": {
+                                        "literalValue": {
+                                            "width": 1366,
+                                            "height": 768
+                                        }
+                                    }
+                                },
+                                "normal": {
+                                    "outputValue": {
+                                        "literalValue": {
+                                            "width": 1366,
+                                            "height": 768
```

Comment:
It looks like this one and the "high" resolution ones are identical... probably a mistake when modifying the transformations... I've tested the pull request and everything looks good - so once this is fixed, it should be ready to go in